test/app.test.js

```diff
@@ -0,0 +1,99 @@
+require('dotenv').config();
+
+const request = require('supertest');
+const app = require('../lib/app');
+const connect = require('../lib/utils/connect');
+const mongoose = require('mongoose');
+const Meme = require('../lib/models/Meme');
+
+describe('memey stuff tests', () => {
+
+  beforeAll(() => {
+    connect();
+  });
+ 
+  beforeEach(() => {
+    return mongoose.connection.dropDatabase();
+  });
+ 
+  afterAll(() => {
+    return mongoose.connection.close();
+  });
+  
+  it('post', () => {
```

Comment:
If you were on a codebase and submitting a PR with tests, it's good to be explicit in what your tests are doing. it('can post a meme')